packages/platform-android/native_modules.gradle

```diff
@@ -0,0 +1,249 @@
+import groovy.json.JsonSlurper
```

Comment:
please add this file `"files"` array in `package.json` so it's published: https://github.com/react-native-community/cli/blob/7d9f8cd96f8e5abc5b7c517d02fb166f9e51b877/packages/platform-android/package.json#L12-L14

Reply:
We forgot about it for iOS 😄 https://github.com/react-native-community/cli/pull/331

Reply:
Done :D